test/integ_tests/test_all_notebooks.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+import os
+import pytest
+
+from testbook import testbook
+from importlib.machinery import SourceFileLoader
+
+# These notebooks have syntax or dependency issues that prevent them from being tested.
+EXCLUDED_NOTEBOOKS = [
+    # These notebooks have cells that have syntax errors
+    "4_Operating_Borealis_beginner_tutorial.ipynb",
+    "bring_your_own_container.ipynb",
+    "qnspsa_with_embedded_simulator.ipynb",
+    # These notebooks have dependency issues
+    "VQE_chemistry_braket.ipynb",
+    "6_Adjoint_gradient_computation.ipynb",
+    # These notebooks are in flux
+    "Using_The_Adjoint_Gradient_Result_Type.ipynb"
```

Comment:
I think these notebooks should be fine now, after updating the BDK, pennylane, simulator versions.

Reply:
Last time I tried to run them they failed for various reasons listed in the comments. Some have cells commented out that lead to syntax errors, and others include openfermion (which has issues with 3.7).

Reply:
The Adjoint Gradient one is really interesting because it doesn't seem to be consistent. Running the notebook twice with the exact same results from a QPU will run differently. We can't mock it out until we prevent the randomness.